<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindw</class>
 <widget class="QMainWindow" name="MainWindw">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>928</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_8">
    <item row="1" column="0">
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_7"/>
     </widget>
    </item>
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="1" column="1">
       <widget class="QScrollArea" name="scrollArea">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>271</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>269</width>
           <height>471</height>
          </rect>
         </property>
         <widget class="QPushButton" name="BtnGraphics">
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>450</y>
            <width>87</width>
            <height>23</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Graphical output</string>
          </property>
         </widget>
         <widget class="QPushButton" name="btnSort">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>330</y>
            <width>249</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>Sort</string>
          </property>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>170</y>
            <width>135</width>
            <height>147</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QGroupBox" name="groupBox_6">
             <property name="title">
              <string>Sort by</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_5">
              <item row="0" column="0">
               <layout class="QFormLayout" name="formLayout">
                <item row="0" column="0">
                 <widget class="QRadioButton" name="amountASC">
                  <property name="text">
                   <string>Price: low to high</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QRadioButton" name="amountDESC">
                  <property name="text">
                   <string>Price: high to low</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QRadioButton" name="dateDESC">
                  <property name="text">
                   <string>Date: new to old</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QRadioButton" name="dateASC">
                  <property name="text">
                   <string>Date: old to new</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>242</width>
            <height>135</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QGroupBox" name="groupBox_5">
             <property name="title">
              <string>Filter your expences</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_4">
              <item row="0" column="0">
               <widget class="QLabel" name="label_16">
                <property name="text">
                 <string>from:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QDateEdit" name="dateFilterFrom"/>
              </item>
              <item row="0" column="2">
               <widget class="QLabel" name="label_17">
                <property name="text">
                 <string>to:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="3">
               <widget class="QDateEdit" name="dateFilterTo"/>
              </item>
              <item row="1" column="0" colspan="3">
               <widget class="QSplitter" name="splitter_12">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <widget class="QSplitter" name="splitter_13">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <widget class="QLabel" name="label_18">
                  <property name="text">
                   <string>cathegory:</string>
                  </property>
                 </widget>
                 <widget class="QComboBox" name="boxCathegoryFilter">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="currentText">
                   <string/>
                  </property>
                 </widget>
                </widget>
                <widget class="QSplitter" name="splitter_14">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <widget class="QLabel" name="label_19">
                  <property name="text">
                   <string>person:</string>
                  </property>
                 </widget>
                 <widget class="QComboBox" name="boxPersonFilter"/>
                </widget>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="btnFilter">
             <property name="text">
              <string>Filter</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>390</y>
            <width>206</width>
            <height>27</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_15">
             <property name="font">
              <font>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Total sum:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="totalSum">
             <property name="font">
              <font>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QTableView" name="tableView">
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QFrame" name="frame_2">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="1">
          <widget class="QLineEdit" name="amount">
           <property name="placeholderText">
            <string>amount</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>cathegory:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QComboBox" name="boxCathegory">
           <property name="currentText">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>person:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QComboBox" name="boxPerson"/>
         </item>
         <item row="0" column="6">
          <widget class="QDateEdit" name="dateAdd"/>
         </item>
         <item row="0" column="7">
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item row="0" column="8">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>270</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>928</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionnew_person"/>
   <addaction name="separator"/>
   <addaction name="actionnew_cathegory"/>
   <addaction name="separator"/>
   <addaction name="actionOpenProject"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="actionDeleteItem"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionnew_person">
   <property name="text">
    <string>new person</string>
   </property>
   <property name="toolTip">
    <string>add new person to your tracker</string>
   </property>
  </action>
  <action name="actionnew_cathegory">
   <property name="text">
    <string>new cathegory</string>
   </property>
   <property name="toolTip">
    <string>add new cathegory</string>
   </property>
  </action>
  <action name="actionOpenProject">
   <property name="text">
    <string>open project</string>
   </property>
   <property name="toolTip">
    <string>open existing project or create new one</string>
   </property>
  </action>
  <action name="actionDeleteItem">
   <property name="text">
    <string>delete item</string>
   </property>
   <property name="toolTip">
    <string>delete item from database</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
